packages/test-runner/src/fixtures.ts

```diff
@@ -74,15 +74,19 @@ class Fixture {
       return;
     for (const name of this.deps) {
       await this.pool.setupFixture(name, config, info);
-      this.pool.instances.get(name).usages.add(this.name);
+      for (const instance of this.pool.instances.get(name).all())
+        instance.usages.add(this.name);
```

Comment:
Different overrides will have different dependencies, so usages should be precise, not merged by the name.